Fetch/Fetch.php

```diff
@@ -461,4 +467,41 @@ private function checkDeep($params)
 
         return is_null($param) ? $this->getConfigBool('deep') : bool($param);
     }
+
+    private function addChildPagesToPage(\Statamic\Data\Pages\Page $page): array
```

Comment:
I can remove the type hints (PHP 7+) if you want.
But as far as I see the Plugin won't run anyway in prior versions

Reply:
At this point I can barely remember what features were added in 7+ 😕 

I know these Return Types are new, but everything else in the Addon should work... I think? Maybe saver to remove them for now, if that's alright.

Reply:
The global method breaks the addon prior PHP 7 because `global` is [a keyword](http://php.net/manual/en/migration70.other-changes.php).
``` php
public function global($handle = null)
```

IMHO, there is no need for compatibility for PHP<7 because those versions are no longer supported and there are real benefits with the new features.

But I can remove them if you want.

Reply:
Ok no problem, leave it as is — I think it's time everyone gets on the PHP 7 bandwagon anyway 😄